src/lib/stores/data-converter-websocket.ts

```diff
@@ -0,0 +1,43 @@
+import { get } from 'svelte/store';
+import WebSocketAsPromised from 'websocket-as-promised';
+import {
+  dataConverterPort,
+  setLastDataConverterFailure,
+} from './data-converter-config';
+
+export const dataConverterWebsocket = (function () {
```

Comment:
Why does this need to be an IIFE? Modules are inherently IIFEs.

Reply:
Stack overflow copy pasta from before vacation didn't fix it after you made this comment last time, will fix.